src/src/tx_main.cpp

```diff
@@ -350,16 +350,13 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
   }
   else
   {
-    if ((millis() > (MSP_PACKET_SEND_INTERVAL + MSPPacketLastSent)) && MSPPacketSendCount)
```

Comment:
We need this.

Reply:
Added. Please double triple check this bit ;)